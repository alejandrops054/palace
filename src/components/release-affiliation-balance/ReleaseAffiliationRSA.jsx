/* Sistema Intelligence 2.0
Módulo Contratos
Versión 1.0
Author: IntregraIT Contratos
Comentarios: xxxxx
*/

import React from "react";
import { Row, Col, Button, Label, Input, Container } from "reactstrap";

function ReleaseAffiliationRSA() {
  return (
    <>
      <Row>
        <Col className="border">
          <span>Remain</span>
          <Row>
            <Col>
              <Row>
                <Label check>
                  <Input type="checkbox" /> Monthly Payment
                </Label>
              </Row>
              <Row>
                <Label check>
                  <Input type="checkbox" /> Down Payment
                </Label>
              </Row>
            </Col>
          </Row>
        </Col>
        <Col className="border">
          <span>Source</span>
          <Row>
            <Col>
              <Row>
                <Label check>
                  <Input type="checkbox" /> All
                </Label>
              </Row>
              <Row>
                <Label check>
                  <Input type="checkbox" /> With Suspense
                </Label>
              </Row>
              <Row>
                <Label check>
                  <Input type="checkbox" /> Without Suspense
                </Label>
              </Row>
            </Col>
          </Row>
        </Col>
        <Col className="border">
          <span>Amounts</span>
          <Row>
            <Col>
              <Row>
                <Label check>
                  Max Balance <Input type="number" /> USD
                </Label>
              </Row>
              <Row>
                <Label check>
                  Exchange rate <Input type="number" />
                </Label>
              </Row>
            </Col>
          </Row>
        </Col>
      </Row>
      <Row className="mt-2">
        <Col lg={{ offset: 9, size: 4 }} md={{ offset: 8, size: 5 }}>
          <Button>Simulate</Button>
          <Button>Process</Button>
          <Button>Cancel</Button>
        </Col>
      </Row>
    </>
  );
}

export default ReleaseAffiliationRSA;
