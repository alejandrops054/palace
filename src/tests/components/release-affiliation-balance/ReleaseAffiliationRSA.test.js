/* Sistema Intelligence 2.0
Módulo Contratos
Versión 1.0
Author: IntregraIT Contratos
Comentarios: xxxxx
*/
import React from 'react';
import '@testing-library/jest-dom'
import { shallow } from 'enzyme'
import ReleaseAffiliationRSA from '../../../components/release-affiliation-balance/ReleaseAffiliationRSA'

describe('ReleaseAffiliationRSA component', () => {
  test('debe mostrar snapshot', () => {

    const wrapper = shallow(<ReleaseAffiliationRSA />)
    expect(wrapper).toMatchSnapshot();
  })

})