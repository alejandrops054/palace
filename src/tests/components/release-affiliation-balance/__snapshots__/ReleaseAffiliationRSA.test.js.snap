// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ReleaseAffiliationRSA component debe mostrar snapshot 1`] = `
<Fragment>
  <Row
    tag="div"
    widths={
      Array [
        "xs",
        "sm",
        "md",
        "lg",
        "xl",
      ]
    }
  >
    <Col
      className="border"
      tag="div"
      widths={
        Array [
          "xs",
          "sm",
          "md",
          "lg",
          "xl",
        ]
      }
    >
      <span>
        Remain
      </span>
      <Row
        tag="div"
        widths={
          Array [
            "xs",
            "sm",
            "md",
            "lg",
            "xl",
          ]
        }
      >
        <Col
          tag="div"
          widths={
            Array [
              "xs",
              "sm",
              "md",
              "lg",
              "xl",
            ]
          }
        >
          <Row
            tag="div"
            widths={
              Array [
                "xs",
                "sm",
                "md",
                "lg",
                "xl",
              ]
            }
          >
            <Label
              check={true}
              tag="label"
              widths={
                Array [
                  "xs",
                  "sm",
                  "md",
                  "lg",
                  "xl",
                ]
              }
            >
              <Input
                type="checkbox"
              />
               Monthly Payment
            </Label>
          </Row>
          <Row
            tag="div"
            widths={
              Array [
                "xs",
                "sm",
                "md",
                "lg",
                "xl",
              ]
            }
          >
            <Label
              check={true}
              tag="label"
              widths={
                Array [
                  "xs",
                  "sm",
                  "md",
                  "lg",
                  "xl",
                ]
              }
            >
              <Input
                type="checkbox"
              />
               Down Payment
            </Label>
          </Row>
        </Col>
      </Row>
    </Col>
    <Col
      className="border"
      tag="div"
      widths={
        Array [
          "xs",
          "sm",
          "md",
          "lg",
          "xl",
        ]
      }
    >
      <span>
        Source
      </span>
      <Row
        tag="div"
        widths={
          Array [
            "xs",
            "sm",
            "md",
            "lg",
            "xl",
          ]
        }
      >
        <Col
          tag="div"
          widths={
            Array [
              "xs",
              "sm",
              "md",
              "lg",
              "xl",
            ]
          }
        >
          <Row
            tag="div"
            widths={
              Array [
                "xs",
                "sm",
                "md",
                "lg",
                "xl",
              ]
            }
          >
            <Label
              check={true}
              tag="label"
              widths={
                Array [
                  "xs",
                  "sm",
                  "md",
                  "lg",
                  "xl",
                ]
              }
            >
              <Input
                type="checkbox"
              />
               All
            </Label>
          </Row>
          <Row
            tag="div"
            widths={
              Array [
                "xs",
                "sm",
                "md",
                "lg",
                "xl",
              ]
            }
          >
            <Label
              check={true}
              tag="label"
              widths={
                Array [
                  "xs",
                  "sm",
                  "md",
                  "lg",
                  "xl",
                ]
              }
            >
              <Input
                type="checkbox"
              />
               With Suspense
            </Label>
          </Row>
          <Row
            tag="div"
            widths={
              Array [
                "xs",
                "sm",
                "md",
                "lg",
                "xl",
              ]
            }
          >
            <Label
              check={true}
              tag="label"
              widths={
                Array [
                  "xs",
                  "sm",
                  "md",
                  "lg",
                  "xl",
                ]
              }
            >
              <Input
                type="checkbox"
              />
               Without Suspense
            </Label>
          </Row>
        </Col>
      </Row>
    </Col>
    <Col
      className="border"
      tag="div"
      widths={
        Array [
          "xs",
          "sm",
          "md",
          "lg",
          "xl",
        ]
      }
    >
      <span>
        Amounts
      </span>
      <Row
        tag="div"
        widths={
          Array [
            "xs",
            "sm",
            "md",
            "lg",
            "xl",
          ]
        }
      >
        <Col
          tag="div"
          widths={
            Array [
              "xs",
              "sm",
              "md",
              "lg",
              "xl",
            ]
          }
        >
          <Row
            tag="div"
            widths={
              Array [
                "xs",
                "sm",
                "md",
                "lg",
                "xl",
              ]
            }
          >
            <Label
              check={true}
              tag="label"
              widths={
                Array [
                  "xs",
                  "sm",
                  "md",
                  "lg",
                  "xl",
                ]
              }
            >
              Max Balance 
              <Input
                type="number"
              />
               USD
            </Label>
          </Row>
          <Row
            tag="div"
            widths={
              Array [
                "xs",
                "sm",
                "md",
                "lg",
                "xl",
              ]
            }
          >
            <Label
              check={true}
              tag="label"
              widths={
                Array [
                  "xs",
                  "sm",
                  "md",
                  "lg",
                  "xl",
                ]
              }
            >
              Exchange rate 
              <Input
                type="number"
              />
            </Label>
          </Row>
        </Col>
      </Row>
    </Col>
  </Row>
  <Row
    className="mt-2"
    tag="div"
    widths={
      Array [
        "xs",
        "sm",
        "md",
        "lg",
        "xl",
      ]
    }
  >
    <Col
      lg={
        Object {
          "offset": 9,
          "size": 4,
        }
      }
      md={
        Object {
          "offset": 8,
          "size": 5,
        }
      }
      tag="div"
      widths={
        Array [
          "xs",
          "sm",
          "md",
          "lg",
          "xl",
        ]
      }
    >
      <Button
        color="secondary"
        tag="button"
      >
        Simulate
      </Button>
      <Button
        color="secondary"
        tag="button"
      >
        Process
      </Button>
      <Button
        color="secondary"
        tag="button"
      >
        Cancel
      </Button>
    </Col>
  </Row>
</Fragment>
`;
